src/DatabaseMain.py

```diff
@@ -0,0 +1,29 @@
+import src.InitializeDatabaseScript as DB
+
+
+if __name__ == "__main__":
+    db = DB.Database()
+    user_name = "postgres"
+    password = "my_secret_password"
+    host = "localhost"
+    port = 5432
+    database_name = "test_db5"
```

Comment:
We may be able to reference the variables used in the ini file for these values

Reply:
Done